tests/features/common/kie-kieserver-common.feature

```diff
@@ -499,10 +499,6 @@ Feature: Kie Server common features
     And file /home/jboss/.m2/settings.xml should contain <url>http://nexus-test.127.0.0.1.nip.ip/nexus/</url>
     And file /home/jboss/.m2/settings.xml should contain <mirrorOf>external:*</mirrorOf>
 
-  Scenario: RHPAM-3517: Update maven to 3.6
-    When container is started with command bash
-    Then run sh -c "mvn --version | sed -n -e 's/^.*Apache //p' | grep 3.6 && echo  all good" in container and check its output for all good
```

Comment:
Wouldn't be better to rather check the current supported version instead of removing the test?

Reply:
I have added already but forgot to remote this one, see: https://github.com/jboss-container-images/jboss-kie-modules/blob/main/tests/features/common/kie-maven-common.feature

Reply:
remove*